workflow/common/common.go

```diff
@@ -159,6 +159,9 @@ const (
 	// EnvAgentPatchRate is the rate that the Argo Agent will patch the Workflow TaskSet
 	EnvAgentPatchRate = "ARGO_AGENT_PATCH_RATE"
 
+	// EnvExecGCWorkers is the number of artifact GC workers for argoexec
+	EnvExecGCWorkers = "ARGO_EXEC_GC_WORKERS"
```

Comment:
imo if its only used for artifact gc perhaps the env name should also carry this information? Just to be more obvious to anyone who is looking at a .env file. 
`ARGO_EXEC_ARTI_GC_WORKERS` ?

Reply:
Nice